.gitignore

```
@@ -60,7 +60,8 @@ third_party/nlbuild-autotools/repo/third_party/autoconf/m4/lt~obsolete.m4
 third_party/nlbuild-autotools/repo/third_party/autoconf/missing
 /tmp/
 
-# C-Lion / IDEA files
+# IDE / editor files
+.vscode/**
 .idea/**
```

nit: alphabetical ordering?
```suggestion
.idea/**
.vscode/**
```